economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+
+The objective of this analysis is to better understand the relationship, if any, between (1) the variable real value of a subsidy received by service-providers/stakers (the \textit{real yield}), and (2) their subsequent, collective decision to increase, decrease or maintain the size of their stake, expressed as a percentage of the total circulating tokens (the \textit{staking rate}). This pertains directly to the primary objective of the NuCypher staking protocol mentioned in section 1 – to maximise the abundance and reliability of network supply – which is reflected in the health\footnote{`Health' in this context simply means the percentage of tokens staked – for example, a network with only 5\% of tokens staked might be described as unhealthy, because it may be highly centralized and/or not have the capacity to deal with demand} and stability of the staking rate. This study employs data sets from two independent sources, (A) Staking Rewards and (B) Yields:
+\\\\
+(A) Staking Rewards provides data on the networks Tezos, Cosmos, Livepeer and Irisnet. The Staking Rewards time series is 4x daily (every 6 hours) and runs from February 2019 (Livepeer), March 2019 (Cosmos, Irisnet) and July 2019 (Tezos) until February 2020 (all four networks). Variable (1) is referred to as the \textit{adjReward}, and variable (2) as the \textit{totalStaked}. 
+\\\\
+(B) Yields provides data on the networks Tezos, Cosmos, and Livepeer. The Yields time series is daily and runs from July 2019 (Tezos \& Cosmos) and September 2019 (Livepeer) until April 2020 (all three networks). Variable (1) is referred to as the \textit{real yield}, and variable (2) as the \textit{staking rate}. 
+\\\\
+For the networks in both data sets, there are discrepancies between equivalent readings on overlapping days. This may be due to different methods of calculating both variables, but it is beyond the scope of this study to reconcile the data or decide which is superior, and so both sets of results should be interpreted with equal weight, or weighted by sample size. It is worth mentioning that these data entries have been calculated by (A) professional data providers whose products depend on data accuracy, and (B) professional staking operators that currently stake in all the networks in question. Thus both data sets certainly reflect \textbf{an} authentic economic reality, if not \textbf{the} economic reality. 
+
+\subsubsection{Relevant system dynamics}\label{dynamics}
+
+In all the networks under examination, when the staking rate increases, the real yield decreases \textbf{immediately} by a similar magnitude, and vice versa. This is because the amount of stake eligible for a subsidy increases, but the total subsidy available at that same timestamp doesn't change (or changes a relatively small amount) - this means there are fewer tokens to go around. This is a known, causal relationship, that we expect to witness when the time lag between the variables is zero. 
```

Comment:
Instead of "fewer tokens to go around" -> "more stakers to split the same total subsidy"

Saying subsidy remains the same, but then stating fewer tokens to go around seems odd - I know what you meant though.


```suggestion
In all the networks under examination, when the staking rate increases, the real yield decreases \textbf{immediately} by a similar magnitude, and vice versa. This is because the amount of stake eligible for a subsidy increases, but the total subsidy available at that same timestamp doesn't change (or changes a relatively small amount) - this means there are more stakers to split the same total subsidy. This is a known, causal relationship, that we expect to witness when the time lag between the variables is zero. 
```